.github/workflows/cmake.yml

```diff
@@ -41,7 +44,16 @@ jobs:
     - name: Create Build Environment
       # Some projects don't allow in-source building, so create a separate build directory
       # We'll use this as our working directory for all subsequent commands
-      run: cmake -E make_directory ${{github.workspace}}/build
+      run: |
+          echo "tzdata tzdata/Areas select America" > ubuntu-preseed.txt
```

Comment:
this should go into ubuntu package install section, messes up the macOS builds

Reply:
I had to move the make directory creation after the package installation to make sure we have CMake available.